app/models/configured-citation-addon.ts

```diff
@@ -48,6 +51,13 @@ export default class ConfiguredCitationAddonModel extends ConfiguredAddonModel {
         });
         return await newInvocation.save();
     }
+
+    @task
+    @waitFor
+    async getSelectedItemName(this: ConfiguredCitationAddonModel) {
+        const response = await taskFor(this.getItemInfo).perform(this.rootFolder);
+        this.rootFolderName = (response.operationResult as Item).itemName;
+    }
```

Comment:
Non-blocking: should we define this back in `configured-addon.ts` while overriding it in `configured-link-addon? In this way, we won't have duplicate code for storage and citation add-ons.